packages/terra-compact-interactive-list/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## 1.2.0 - (January 26, 2024)
```

Comment:
Why is there no changelog entry for the docs update.

Reply:
It's in another changelog for `terra-framework-docs`. The changes in `compact-interactive-list` package are not user facing (only tests added).